### PR TITLE
Update app_accuracy.tex, minor edits

### PR DESCRIPTION
**Pull request by hdquemada**
### Description

Fixes # (issue)

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] This change requires a documentation update
- [x] Housekeeping

### How Has This Been Tested?

**Test Configuration**:
* Operating system: <Name, version number>
* Graphics Card: <Manufacturer (likely Intel, NVidia, AMD?), Model (HD, Geforce, Radeon..., with model number), driver version?>

## Checklist:

- [ ] My code follows the [code style](http://stellarium.org/doc/head/codingStyle.html) of this project.
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (header file)
- [ ] I have updated the respective chapter in the Stellarium User Guide
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: guide/app_accuracy.tex [hunk 1]
```diff
@@ -33,9 +33,9 @@ \chapter{Accuracy}
 the Solar gravity field, may still be required for critical
 observations of stellar occultations by asteroids, grazing
 occultations of stars by the Moon or similar things. Stellarium does
-not compute these effects, and is definitely not capable for detailed
```
Comment (gzotti) [inline guide/app_accuracy.tex:36]: OK, if we are changing something, admittedly the program is not capable, humans may be. So "not usable for..."

File: guide/app_accuracy.tex [hunk 2]
```diff
@@ -33,9 +33,9 @@ \chapter{Accuracy}
...
+not compute these effects, and is definitely not capable of detailed
 planning of your spacecraft missions to the planets.
-There are more specialized programs for these topics, please use them.
+There are more specialized programs for these topics. Please use them.
```
Comment (gzotti) [inline guide/app_accuracy.tex:39]: For a particular large group of readers, somebody recommended using short sentences. I wanted to be understood. ;-)

File: guide/app_accuracy.tex [hunk 3]
```diff
@@ -63,7 +63,7 @@ \section{Stellar Proper Motion}
 please note that the computation only takes the linear components $\Delta\alpha$, $\Delta\delta$ into account. 
 For times far from today, a true 3D computation would be required, 
 so that also changes in distance (and thus, brightness) could be simulated. 
-Still, the simulation here should give a good impression which close and/or fast stars
```
Comment (gzotti) [inline guide/app_accuracy.tex:66]: No, which stars in detail, not just a general observation.

File: guide/app_accuracy.tex [hunk 4]
```diff
@@ -125,7 +125,7 @@ \section{Minor Bodies}
 
 Positions for the Minor Bodies (Dwarf Planets, Asteroids, Comets) are
 computed with standard algorithms found in astronomical text
-books. The generally used method of orbital elements allows to compute
+books. The generally used method of orbital elements allows the user to compute
```
Comment (gzotti) [inline guide/app_accuracy.tex:128]: probably "allows computing". Not just the user of our program, but anybody who uses the scientific model.

File: guide/app_accuracy.tex [hunk 5]
```diff
@@ -164,7 +164,7 @@ \section{Eclipses}
 
 The moon's motion is very complicated, and eclipse computations can be
 tricky.  One aspect which every student of history, prehistory and
-archaeology should know but as it seems not every does, is at least a
+archaeology should know, but as it seems not every does, is at least a
```
Comment (gzotti) [inline guide/app_accuracy.tex:167]: OK, it's a complete insertion.

File: guide/app_accuracy.tex [hunk 6]
```diff
@@ -202,17 +202,17 @@ \section{The Calendar}
   \S189]{Ginzel:ChronologieII}. Be careful if you are working in
 Augustean and earlier times in the Roman world, maybe even related to
 named dates in the pre-Julian Roman calendar: Stellarium, also its
-Calendars plugin (see section \ref{sec:plugin:Calendars}) does not
+Calendars plugin (see section \ref{sec:plugin:Calendars}), does not
```
Comment (gzotti) [inline guide/app_accuracy.tex:205]: OK.

File: guide/app_accuracy.tex [hunk 7]
```diff
@@ -253,7 +253,7 @@ \section{Comparison to Reference Data}
 provided by JPL's solutions DE440/DE441 in J2000.0 equatorial
 coordinates, including JPL's corrections for nutation, light time,
 aberration, light deflection in the Solar gravity field, and other
-relativistic effects. The right part provides the according values
```
Comment (gzotti) [inline guide/app_accuracy.tex:256]: If you think this is in any way better, we can say "respective values". I want to express "these are the results of Stellarium 0.21.2&beta; put next to/adjacent/side-to-side  the results of JPL Horizons for the same input dates."

**Review by gzotti — CHANGES_REQUESTED** [review]
Hi! Thanks for helping. We may take over a few changes, but not all. Will do that on Saturday.

**Comment by hdquemada** [comment]
Great! Glad to help. Of course, use only what you think is helpful. I will be going through the other parts of the guide as well, so I hope it will be of use.

---
Hector

**Pull request closed without merging**